test/utils/fixture/devices.go

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2022 Red Hat, Inc.
```

Comment:
let's review the copyright year per the conversartion in the comments (now I need to review if the correct choice is 2021 or 2022 :) )

Reply:
ok, this is new code so 2022 is fine

Reply:
thanks for your input, may you please tell why 2022 and not 2021 (the current year)? I assume this will be available in 4.12 GA, right?